css/styles.css

```diff
@@ -2,4 +2,23 @@
   font-size: 50px;
   margin: auto;
   width: 500px;
+  text-align: center;
+}
+
+#totalscore {
```

Comment:
`#score` and `#totalscore` are the same. Why not making a class called .scores and then style that?